### PR TITLE
Add rick to start screen, fix customer count bug!

### PR DESCRIPTION
**Pull request by ConorMcPersonal**
Was adding Rick to start screen, but noticed that customer count never increased!!

### CODE REVIEW

File: SplitFiles/base.c [hunk 1]
```diff
@@ -17,7 +17,7 @@ GameParameters* get_game_parameters() {
     params->breadBin = NULL;
     params->reputation = 1000; //Start with 1000 rep
     params->hotelOpen = 1;     //Open by default
-    params->maxCustomers = 6;  //Start with a reasonable cuustomeer queue, goes up each day
+    params->maxCustomers = 4;  //Start with a reasonable cuustomeer queue, goes up at start of each day
```
Comment (ConorMcPersonal) [inline SplitFiles/base.c:20]: This actually results in a starting count of 5 now - was always 6 customer max before

File: SplitFiles/game.c [hunk 2]
```diff
@@ -159,8 +159,20 @@ int main_game( int hiScore )
...
+    mplayer->add_music(mplayer, TUNE_RICKROLL, 1);
+    int rando = 0;
+    while (in_inkey() == 0) {
+      mplayer->play(mplayer);
+      rando += 1;
+    }
+    while (in_inkey() != 0) {
+      //mplayer->play(mplayer);
+      rando += 17;
+    }
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:174]: Rick until you press (and release) a key

File: SplitFiles/game.c [hunk 3]
```diff
@@ -177,6 +189,8 @@ int main_game( int hiScore )
         params->hotelOpen = 1;
 
         game_day += 1;
+        //Actually increase the number of customers
+        params->maxCustomers = MIN(params->maxCustomers + 1, 10);
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:193]: Actually increase the number of customers - max is 10 (screen limitations)

File: SplitFiles/game.c [hunk 4]
```diff
@@ -223,8 +237,10 @@ int main_game( int hiScore )
         }
 
         if (params->gameOverFlag == 0) {
-          printf("\x10\x34We're opening up to more\n"
-                 "customers than ever before.\x10\x30\n");
+          if (params ->maxCustomers < CUSTOMER_MAX) {
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:240]: Don't say this unless it's true

File: SplitFiles/game.c [hunk 5]
```diff
@@ -177,6 +189,8 @@ int main_game( int hiScore )
         params->hotelOpen = 1;
 
         game_day += 1;
+        //Actually increase the number of customers
+        params->maxCustomers = MIN(params->maxCustomers + 1, CUSTOMER_MAX);
```
Comment (ConorMcPersonal) [inline SplitFiles/game.c:193]: Can't believe this wasn't in the code!

**Review by mcmenaminadrian — APPROVED** [review]
See my suggested addition.

My actual idea was to have rick play over the "Toast Host" attractor, but it works this way too.

If you are happy with my addition please merge, otherwise kill/edit it and merge

**Comment by ConorMcPersonal** [comment]
Easy to move it to the attractor - it's a bit much if you're trying to
actually read the text!

**Pull request merged**